results/proto/v1alpha1/api.proto

```diff
@@ -180,4 +180,9 @@ message ListResultsRequest {
 message ListResultsResponse {
   repeated Result results = 1;
   string next_page_token = 2;
+}
+
+message ListPageIdentifier {
+	string resultName = 1;
+  string filter = 2;
```

Comment:
We should break this message out into its own `internal` proto package so that it's not included in the API definition.